driver_cpl/cime_config/config_component.xml

```diff
@@ -2974,14 +2974,6 @@
     <desc>Whether to clean the test after it is built/run</desc>
   </entry>
 
-  <entry id="CCSM_BASELINE">
```

Comment:
CCSM_BASELINE has been replaced in the code by BASELINE_ROOT - I'll finish this cleanup by replacing the rest of the cases of CCSM_BASELINE in cime.